modules/channel-vonage/src/backend/client.ts

```diff
@@ -115,6 +116,7 @@ export class VonageClient {
       new VonageVideoRenderer(),
       new VonageTemplateRenderer(),
       new VonageMediaTemplateRenderer(),
+      new VonageFileRenderer(),
       new VonageChoicesRenderer()
```

Comment:
I made a small fix here. The `priority` property doesn't actually do anything on renderers at the moment so you have to order them yourself in the array (it used to be the we collected from modules and ordered there but this has since been removed).